sonar-python-plugin/src/test/java/org/sonar/plugins/python/IPynbRuleRepositoryTest.java

```diff
@@ -43,7 +43,7 @@ public void createRulesTest() throws IOException {
 
     assertThat(repository).isNotNull();
     assertThat(repository.language()).isEqualTo("ipynb");
-    assertThat(repository.name()).isEqualTo("SonarQube");
+    assertThat(repository.name()).isEqualTo("Sonar");
```

Comment:
I'm a bit surprised by the change of repository name. I would suggest to check if everything is still working on SonarLint analyzing a Jupyter Notebook file

Reply:
checked, looks good